source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,188 @@
+
+### Changes in Ember Data 3.13
+
+#### New Features (0)
+
+No new features introduced in Ember Data 3.13.
+
+#### Deprecations (0)
+
+No new deprecations introduced in Ember Data 3.13.
+
+For more details on changes in Ember Data 3.13, please review the
+[Ember Data 3.13.0 release page](https://github.com/emberjs/data/releases/tag/v3.13.0).
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI easily using the ember-cli-update project:
+
+```bash
+npm install -g ember-cli-update
+ember-cli-update
+```
+
+This utility will help you to update your app or add-on to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the github README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and Ember Data, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or Ember Data by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 3.13
+
+No new features in Ember CLI 3.13.
+
+---
+
+For more details on the changes in Ember CLI 3.13 and detailed upgrade
+instructions, please review the [Ember CLI  3.13.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v3.13.0).
+
+## Ember 3.12 is an LTS Release
+
+If you are upgrading from the previous LTS version (3.8) to 3.12 then you are getting access to these features:
+
+* `router` service 
+* Support for native classes
+* The `array` helper
+* Element Modifier Manager (and modifiers in general)
+* Nested angle bracket component invocation
+* Angle bracket invocation for built-in components
+* Support for native decorators
+* The `{{fn}}` and `{{on}}` modifiers
+
+## 3.14 Beta
+
+This release kicks off the 3.14 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+Among other things, the [`@model` named argument](https://github.com/emberjs/rfcs/blob/master/text/0523-model-argument-for-route-templates.md) will be available to route templates. While not officially an "Octane feature", this feature will available for the first time in the 3.14 release, which coincides with the Octane release. Since the Octane guides and documentation are targeting the same version, you will see them using this feature, but please not that the feature is not available in the 3.13 Octane preview.
```

Comment:
```suggestion
Among other things, the [`@model` named argument](https://github.com/emberjs/rfcs/blob/master/text/0523-model-argument-for-route-templates.md) will be available to route templates. While not officially an "Octane feature", this feature will available for the first time in the 3.14 release, which coincides with the Octane release. Since the Octane guides and documentation are targeting the same version, you will see them using this feature, but please note that the feature is not available in the 3.13 Octane preview.
```